src/includes/class-alma-wc-payment-gateway.php

```diff
@@ -862,16 +862,15 @@ public function woocommerce_available_payment_gateways( $_available_gateways ) {
 		}
 
 		$new_available_gateways = array();
-		foreach ( $_available_gateways as $gateway ) {
+		foreach ( $_available_gateways as $key => $gateway ) {
+			$new_available_gateways[ $key ] = $gateway;
 
-			if ( 'alma' !== $gateway->id ) {
-				continue;
+			if ( 'alma' === $gateway->id ) {
+				$new_available_gateways = array_merge( $new_available_gateways, $this->build_new_available_gateways( $gateway ) );
```

Comment:
don't forget to talk of this refactor in your git history (commit) or in the changelog to keep it in mind if a but occurs at this step.